CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Export
 Service to export YourLoops data in selected formats, either csv, json or xls.
 
+## 1.6.0 - 2021-05-13
+### Changed
+- YLP-536 Listen to SIGTERM & SIGINT signals & quickly shutdown when there is no export in progress.
+### Engineering
+- YLP-537 Update to nodejs v12
+- Replace Travis with Jenkins
```

Comment:
and "Add OpenAPI doc"?

Reply:
`Mode flemme: ON`
Not quite, I have done just the /status
I left the other to another PR.
`Mode flemme: OFF`

Reply:
ok, do we have a Jira ticket for that at least?